web/source/osk/embedded/keytip.ts

```diff
@@ -20,6 +20,8 @@ namespace com.keyman.osk.embedded {
     show(key: KeyElement, on: boolean, vkbd: VisualKeyboard) {
       let util = com.keyman.singleton.util;
 
+      //console.log('KeyTip.show', key ? key.id : null, on);
```

Comment:
Is it your intent to insert this comment here?

Reply:
Yeah, will remove the logging comments before merge.